app/services/CheckoutService.scala

```diff
@@ -39,12 +40,26 @@ class CheckoutService(identityService: IdentityService, salesforceService: Sales
         identityService.registerGuest(subscriptionData.personalData)
       }
 
+    userOrElseRegisterGuest.onComplete(t => Logger.info("userOrElseRegisterGuest result "+t))
```

Comment:
is this logging statement still needed? If so, perhaps we should put a more descriptive message.